import java.lang.Exception.*;
class Program2
{
	method1()
	{
	  System.out.println("Exception");
	}
	public static void main(String args[])
	{
	    try
	   {
         program2= (null);
         s.method1();
	   }
	   catch(NullPointerException e){System.out.println(e);}
	   System.out.println("rest of the code");
	}
}